### ISSUE TITLE
Managed Ruleset version support

### ISSUE THREAD

**Issue by sachinc-ops** [umotif-public/terraform-aws-waf-webaclv2#54]
**What is the current behavior?**
We cannot pin AWS Managed Rules to a specific version.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
```      
     managed_rule_group_statement = {
        name        = "AWSManagedRulesPHPRuleSet"
        vendor_name = "AWS"
        version     = "Version_1.0"
      }
```
The '_version_' parameter is currently not supported in this module.

**What is the expected behavior?**
Sometimes it's important to apply or keep specific version because of breaking changes from Amazon.

**Software versions?**
Terraform v1.1.6
module 3.7.0
AWS provider v4.13.0

**Comment by Ohid25** [comment]
I've created this #55 PR with this request.

**Comment by Ohid25** [comment]
Now available in [3.7.3](https://github.com/umotif-public/terraform-aws-waf-webaclv2/releases/tag/3.7.3) release. Thank you for raising a feature request!
